go/service/chat_local.go

```diff
@@ -59,12 +59,98 @@ func (h *chatLocalHandler) GetThreadLocal(ctx context.Context, arg keybase1.GetT
 	return h.unboxThread(ctx, boxed, arg.ConversationID)
 }
 
+func retryUpToNTimesUntilNoError(n int, action func() error) (err error) {
```

Comment:
Just gives the retry a better chance to succeed (like if it failed due to Gregor restarting, waiting would allow this to succeed when it came back up. If you just blast 3 attempts out as fast you can you might miss it).